yasa/hypno.py

```diff
@@ -97,12 +98,19 @@ class Hypnogram:
     11    SLEEP
     Name: Stage, dtype: object
 
+    >>> # Number of epochs in the hypnogram
     >>> hyp.n_epochs
     12
 
+    >>> # Total duration of the hypnogram, in minutes (12 epochs * 30 seconds = 6 minutes)
+    >>> hyp.duration
+    6.0
+
+    >>> # Default mapping from strings to integers. Can be changed with `hyp.mapping = {}`
     >>> hyp.mapping
     {'WAKE': 0, 'SLEEP': 1, 'ART': -1, 'UNS': -2}
 
+    >>> # Get the hypnogram with integers values
```

Comment:
a typo so minor you don't even really have to change it (integers --> integer)

But _if_ you do go in, I would add "Get the hypnogram Series integer values". Something that indicates it's the Series and not actually another Hypnogram instance. I found this distinction only mildly confusing at first,  because if you "look" at the Hypnogram object interactively, it looks just like a Series but it's not. I like this feature, but when the user returns `as_int()` it's actually more like `Hypnogram.hypno` than `Hypnogram`. Again, don't even need to add this if you just wanna leave it.

Reply:
Good point. Hmm, do you think that `as_int` should return a numpy.array to avoid any confusion?

Reply:
No I like it as a Series! The pandas Series is really useful, plus it has the timestamps on it and such. I think it's 100% worth the (very minor) effort to figure out what's going on.